worlds/animal_well/bean_patcher.py

```diff
@@ -1,8 +1,19 @@
+import string
+import pymem
+import pymem.ressources.kernel32
+import ctypes
+
 from time import time
 from typing import List, Optional, Callable, Any, Awaitable, Dict
 
 from .patch import *
 
+def base36(n):
+    out = []
+    while n > 0:
+        n, r = divmod(n, 36)
+        out.append((string.digits + string.ascii_lowercase)[r])
+    return(''.join(reversed(out)))
 
```

Comment:
```suggestion


def base36(n):
    out = []
    while n > 0:
        n, r = divmod(n, 36)
        out.append((string.digits + string.ascii_lowercase)[r])
    return(''.join(reversed(out)))


```
2 newlines between functions/classes (unless it's in a class)